openmetadata-ui/src/main/resources/ui/src/components/MyAssetStats/MyAssetStats.component.tsx

```diff
@@ -112,50 +111,48 @@ const MyAssetStats: FunctionComponent<MyAssetStatsProps> = ({
   );
 
   return (
-    <div className="ant-entity-card">
-      <Card
-        data-testid="data-summary-container"
-        id="assetStatsCount"
-        style={leftPanelAntCardStyle}>
-        <EntityListSkeleton
-          isCount
-          isLabel
-          isSelect
-          loading={Boolean(entityCountLoading)}>
-          <>
-            {Object.values(dataSummary).map((data, index) => (
-              <div
-                className="tw-flex tw-items-center tw-justify-between"
-                data-testid={`${data.dataTestId}-summary`}
-                key={index}>
-                <div className="tw-flex">
-                  <SVGIcons
-                    alt="icon"
-                    className="tw-h-4 tw-w-4 tw-self-center"
-                    icon={data.icon}
-                  />
-                  {data.link ? (
-                    <Link
-                      className="tw-font-medium hover:tw-text-primary-hover hover:tw-underline"
-                      data-testid={data.dataTestId}
-                      to={data.link}>
-                      <Button
-                        className="tw-text-grey-body hover:tw-text-primary-hover hover:tw-underline"
-                        type="text">
-                        {data.data}
-                      </Button>
-                    </Link>
-                  ) : (
-                    <p className="tw-text-grey-body tw-pl-2">{data.data}</p>
-                  )}
-                </div>
-                {!isNil(data.count) && getCountBadge(data.count, '', false)}
+    <Card
+      className="panel-shadow-color"
```

Comment:
Let's add this style to `card.less`

Reply:
https://github.com/open-metadata/OpenMetadata/pull/10391#discussion_r1124037821